src/WebJobs.Script.Scaling/WorkerStatus.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    public enum WorkerStatus
+    {
+        /// <summary>
+        /// This worker may be removed
+        /// </summary>
+        Free = 0,
+
+        /// <summary>
+        /// This worker handles load normally
+        /// </summary>
+        Normal,
+
+        /// <summary>
+        /// This worker is busy
+        /// </summary>
+        Busy
```

Comment:
I'd say that normal for a worker to be busy - do you think this should be called something like Overloaded or something indicating that it is too busy and needs help? Just a naming/clarity question.